### PR TITLE
updating to release 4.3.0

### PR DESCRIPTION
**Pull request by haraldschilly**

### CODE REVIEW

**Comment by haraldschilly** [comment]
I'm new to conda forge, so no idea how this really works. To give reviewers some background, this PR here is motivated by embedded plots by plotly. E.g. this ticket here gives some motivation why updating nbformat is a good idea: https://github.com/plotly/plotly.py/issues/692#issuecomment-287007717

**Comment by jakirkham** [comment]
No worries. This is basically fine. Did you check if there were any dependency changes?

Edit: Looks like there aren't.

**Comment by haraldschilly** [comment]
No, but good idea, I'll have a look.

**Comment by haraldschilly** [comment]
All fine, setup.py wasn't touched since october last year. So what's over there is exactly the same as it is here in the yaml file.

**Comment by jakirkham** [comment]
Admittedly, this is our bad for not doing this, but could you please add `license_file: COPYING.md` to `about` as well? That will include the license file for people downloading our package.

**Comment by haraldschilly** [comment]
ok, build is underway

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,7 +36,8 @@ test:
 about:
   home: http://jupyter.org
   license: BSD 3-clause
-  summary: the base implementation of the Jupyter Notebook format
+  summary: Reference implementation of the Jupyter Notebook format
```
Comment (jakirkham) [inline recipe/meta.yaml]: Seems it has been simplified to, "The Jupyter Notebook format".

ref: https://github.com/jupyter/nbformat/blob/4.3.0/setup.py#L63

Comment (haraldschilly) [reply recipe/meta.yaml]: I'll change it. Funny thing is, I took this from the github repo's description line.

Comment (jakirkham) [reply recipe/meta.yaml]: Hmm...yeah, we have normally used `description` from `setup.py` for Python packages, but obviously that doesn't work for non-Python packages. So we recourse to other strategies. Probably we should document some best practices about metadata. In any event, thanks for updating this.

**Pull request merged**